test/images/scripts/candlepin.bootstrap

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (C) 2016 Red Hat Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA.
+
+BASE=$(dirname $0)
+$BASE/virt-builder-fedora "$1" fedora-22 x86_64
```

Comment:
Sadly the candlepin docs use fedora 22 and my attempts to use 23 or 24 failed. I didn't want to migrate their setup to a newer fedora.